apps/app/src/components/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import { Text, TouchableOpacity } from "react-native";
+  
+  interface ButtonProps {
+    title: string;
+    onPress: () => void;
+  }
+  
+  export const Button = ({ title, onPress }: ButtonProps) => {
+    return (
+      <TouchableOpacity
+        onPress={onPress}
+        className="bg-galaxy-dark-blue items-center rounded-3xl px-20 py-4"
+      >
+        <Text className="text-white font-bold">{title}</Text>
+      </TouchableOpacity>
```

Comment:
**question (non-blocking):** What's the reason for a [TouchableOpacity](https://reactnative.dev/docs/touchableopacity) instead of a [Button](https://reactnative.dev/docs/button) or [Pressable](https://reactnative.dev/docs/pressable)?

Reply:
The `Button` core component does not have `classname` props and thus I cannot add styles with nativewind.
The choose between `TouchableOpacity` instead of `Pressable` is because when you press down `TouchableOpacity`, the opacity of the wrapped view is decreased, which generates a better UX response since the user can note the press down worked.